grafana/scylla-developer.master.template.json

```diff
@@ -0,0 +1,1183 @@
+{
+    "dashboard": {
+        "class": "dashboard",
+        "uid": "developer-master",
+        "originalTitle": "Scylla Developer help",
+        "rows": [
+            {
+                "class": "logo_row"
+            },
+            {
+                "class": "row",
+                "panels": [
+                    {
+                        "class": "single_stat_panel",
+                        "targets": [
+                            {
+                                "expr": "count(up{job=\"scylla\", cluster=~\"$cluster|$^\"})",
+                                "intervalFactor": 1,
+                                "legendFormat": "Total Nodes",
+                                "refId": "A",
+                                "step": 1
+                            }
+                        ],
+                        "title": "Total Nodes"
+                    },
+                    {
+                        "class": "single_stat_panel_fail",
+                        "targets": [
+                            {
+                                "expr": "count(scrape_samples_scraped{job=\"scylla\", cluster=~\"$cluster|$^\"}==0) OR vector(0)",
```

Comment:
what is scrape_samples_scraped?

Reply:
when non 0 it means that there was an answer from a node, this how we decide if a node is reachable